app/controllers/publications/v2/monthly_statistics_controller.rb

```diff
@@ -3,34 +3,30 @@ module V2
     class MonthlyStatisticsController < ApplicationController
       def show
         @presenter = MonthlyStatisticsPresenter.new(current_report)
-        @csv_export_types_and_sizes = calculate_download_sizes(@presenter)
       end
 
-      # TODO: Downloads
       def download
-        # export_type = params[:export_type]
-        # export_filename = "#{export_type}-#{params[:month]}.csv"
-        # raw_data = current_report.statistics[export_type]
-        # header_row = raw_data['rows'].first.keys
-        # data = SafeCSV.generate(raw_data['rows'].map(&:values), header_row)
-        # send_data data, filename: export_filename, disposition: :attachment
-      end
+        return not_found unless csv_exists?
+
+        export_filename = "#{export_type}-#{params[:month]}.csv"
```

Comment:
Worth putting `params[:month]` in a method to be consistent with `export_type`?

Reply:
I'm not sure if this is possible but could there be a scenario where you get here and `params[:month]` is blank?

Reasoning: `csv_exists?` -> `current_report_csv` -> `current_report` method that returns something `if params[:month].blank?`

Reply:
No because when adding the csv link we are adding the month, and if no month is passed current report will return record not found (rendering a 404 page).

Although about params and redirects we will do on the following card: https://trello.com/c/xfHhFuVU/894-monthly-statistics-reactivate-the-monthly-statistics-page

Reply:
You can also replace a few `params[:month]` in the same file, `current_report` method.

Reply:
> You can also replace a few params[:month] in the same file, current_report method.

I should had said , the current_report method will be deleted on the next PR (which I want to delete so much!) so I don't need to refactor for something that will be deleted straight away..